utils/auth.py

```
@@ -0,0 +1,33 @@
+    return ph.hash(password)
+
+def verify_password(plain_password: str, hashed_password: str) -> bool:
+
+    try:
+        return ph.verify(hashed_password, plain_password)
+    except VerifyMismatchError:
+        return False
+
+def create_access_token(data: dict):
+    to_encode = data.copy()
+    expire = datetime.utcnow() + timedelta(minutes=ACCESS_TOKEN_EXPIRE_MINUTES)
```

Ruff (0.14.11)

`datetime.datetime.utcnow()` used

(DTZ003)